openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -715,6 +718,13 @@ port_INLINE bool ieee154e_processIEs(OpenQueueEntry_t* pkt, uint16_t* lenIE) {
             // at this point, ASN and frame length are known
             // the current slotoffset can be inferred
             ieee154e_syncSlotOffset();
+            schedule_syncSlotOffset(ieee154e_vars.slotOffset);

```

Comment:
ieee154e_syncSlotOffset() is in charge of matching the slotoffset to the ASN according to the slotframe length, so it updates ieee154e_vars.slotOffset. That parameter is then passed to schedul_syncSlotOffset, which instead matches the slotOffset to the schedule.